### PR TITLE
fix(app): add support for labware on modules in run log

### PR DESCRIPTION
**Pull request by smb2268**
# Overview

Add support for labware on modules in run log
Screen Shot 2022-08-02 at 10 17 15 AM

# Changelog

Use util `getSlotLabwareName` that parses through all commands to output the correct labware display name and slot for all labware, even ones sitting on top of modules

# Review requests

Test this out!
# Risk assessment

Low

### CODE REVIEW

File: app/src/organisms/Devices/hooks/useLabwareRenderInfoForRunById.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export function useLabwareRenderInfoForRunById(
   )
   const storedProtocolAnalysis = useStoredProtocolAnalysis(runId)
   const protocolData = robotProtocolAnalysis ?? storedProtocolAnalysis
-
+  console.log(protocolData)
```
Comment (shlokamin) [inline app/src/organisms/Devices/hooks/useLabwareRenderInfoForRunById.ts]: leftover console log

Comment (jerader) [inline app/src/organisms/Devices/hooks/useLabwareRenderInfoForRunById.ts]:
```suggestion
```

**Review by jerader — APPROVED** [review]
this lgtm, fixes the white screen 😃

**Review by shlokamin — APPROVED** [review]
lgtm! i think it would be a good idea to move `getSlotLabwareName` into a shared util, but other than that this is great. thanks sarah!

**Pull request merged**